### PR TITLE
installation: extra dependencies for db drivers

### PR DESCRIPTION
**Pull request by jirikuncar**
* Adds extra requirements groups that proxy dependencies for
  *Invenio-DB*.  (addresses #3566)

Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

### CODE REVIEW

**Comment by tiborsimko** [comment]
> "proxy dependencies in Invenio-DB"

proxy dependencies for Invenio-DB

> "(closes #3565) (addresses #3566)"

Keep the closes (linking to the issue), remove the addresses (to an "unrelated" PR that simply "generated" that issue).

**Comment by jirikuncar** [comment]
> simply "generated" that issue

That's why I have added :wink:

File: setup.py [hunk 1]
```diff
@@ -60,10 +62,16 @@
     ],
     'utils': [
         'invenio-mail>=1.0.0a1,<1.1.0',
-        'invenio-db>=1.0.0a5,<1.1.0',
+        'invenio-db' + invenio_db_version,
```
Comment (jirikuncar) [inline setup.py]: This one should be probably removed and postgresql should be added to aliases.

**Comment by tiborsimko** [comment]
LGTM but please rebase against latest master.

**Pull request merged**